scripts/sources.seed.sh

```diff
@@ -4,7 +4,46 @@
 #
 BASEDIR=$(dirname $0)
 source $BASEDIR/sources.conf
-git clone https://github.com/MikelAlejoBR/sources-database-populator $BASEDIR/sources-database-populator
-pushd $BASEDIR/sources-database-populator
-go run main.go
-popd
+
+if [[ -z "$ARN_ROLE" ]]; then
+	echo "ARN_ROLE must be defined in sources.local.conf!"
+	exit 1
+fi
+
+echo "Creating $ARN_ROLE with account_id $ACCOUNT_ID org_id $ORG_ID"
+IDENTITY=$($BASEDIR/identity_header.sh $ACCOUNT_ID $ORG_ID)
```

Comment:
I am getting an error, I am wondering if it just me or for someone else as well: 
```
./scripts/sources.seed.sh: line 14: /identity_header.sh: No such file or directory
{"errors":[{"detail":"Internal Server Error: account number or OrgId not present in x-rh-identity","status":"500"}]}
```

Reply:
Hmmm it worked for me:

```
[lzap@nuc provisioning-backend]$ scripts/sources.seed.sh
Creating arn:aws:iam::12334455655:role/Provisioning1 with account_id 13 org_id 000013
{"sources":[{"id":"1","created_at":"2022-07-18T09:31:40+02:00","updated_at":"2022-07-18T09:31:40+02:00","name":"Amazon source","uid":"c770fbfd-daa7-453a-ac6e-8208cd2ed68a","app_creation_workflow":"manual_configuration","source_type_id":"4"}],"applications":[{"id":"1","created_at":"2022-07-18T09:31:40+02:00","updated_at":"2022-07-18T09:31:40+02:00","source_id":"1","application_type_id":"3"}],"endpoints":[],"authentications":[{"id":"1","authtype":"provisioning-arn","username":"arn:aws:iam::569221671538:role/Provisioning1","resource_type":"Application","resource_id":"1"}]}

[lzap@nuc provisioning-backend]$ scripts/sources.seed.sh
Creating arn:aws:iam::234234234524:role/Provisioning1 with account_id 13 org_id 000013
{"errors":[{"detail":"Internal Server Error: name already exists in tenant","status":"500"}]}
```

Can you investigate? It looks like it cannot find `identity_header.sh` in your case. The script tries to find it in the same directory as the script you executed. Therefore you need to have `scripts/identity_header.sh`. Does it also have executable permissions? Oh by the way do you have ACCOUNT_ID and ORG_ID variables set? You need to have them. They are set in the `sources.conf` by default.

Reply:
So it is executable and I have copied the sources.conf to sources.local.conf and pasted my arn.

Reply:
Can you paste the full error please?

Reply:
Replied on Slack @lzap

Reply:
For the record, sources.conf sources sources.local.conf and if you copied that file it causes endless loop of sourcing itself leading to OOM kill of bash.